src/app/header/header.component.html

```diff
@@ -0,0 +1,69 @@
+<!-- <header *ngIf="isUserLoggedIn$ | async" class="header">
+    <nav class="nav">
+        <ul class="nav__list">
+            <li class="nav__item">
+                <a routerLink="/" class="nav__link">
+                    <img class="nav__icon" src="../../assets/icon.svg">
+                </a>
+            </li>
+            <li class="nav__item">
```

Comment:
May be we can create here an array with values `{value, label}` and use loop here